docs/docs/building.md

```diff
@@ -27,89 +26,63 @@ contract MyContract is Migratable {
+The `create` command takes an optional `--init` flag to call the initialization function after creating the contract, while the `--args` flag is to pass arguments to it. This way, we are initializing our contract with `42` as the value of the `x` state variable.
 
-```
-zos create MyContract --init initialize --args 42 --network development
-```
 
-After these simple steps, your upgradeable application is now on-chain!
+After these simple steps, our upgradeable application is now on-chain!
 
-## Upgrading your contracts
+## Upgrading our contract
 
-If, at a later stage, you want to upgrade your smart contracts' code in order to fix a bug or add a new feature, you can do it seamlessly using ZeppelinOS. 
+If, at a later stage, we want to upgrade our smart contract code in order to fix a bug or add a new feature, we can do it seamlessly using ZeppelinOS. 
 
-Note that while ZeppelinOS supports arbitrary changes in functionality, you will need to preserve all variables that appear in prior versions of your contracts, declaring any new variables below the already existing ones. You can find details on this in the [Advanced topics](advanced.md) page. 
+> **Note**: while ZeppelinOS supports arbitrary changes in functionality, you will need to preserve all variables that appear in prior versions of your contracts, declaring any new variables below the already existing ones. You can find details on this in the [advanced topics](advanced.md) page. 
 
-Once you've made the desired changes to your contracts, recompile them and push them to the network:
+Once we have made the desired changes to our contracts, we need to push them to the network:
 
-```
-npx truffle compile
+```sh
 zos push --network <network>
 ```
 
-Then, upgrade the already deployed contracts:
-
-```
-zos upgrade <contract_name_1> --network <network>
-zos upgrade <contract_name_2> --network <network>
-...
-zos upgrade <contract_name_n> --network <network>
-```
+> **Note**: bear in mind that the `push` command of the ZeppelinOS CLI also compiles our contracts. To avoid this, use the `--skip-compile` flag.
 
-In our simple example:
+Finally, let's upgrade the already deployed contract:
 
+```sh
+zos upgrade MyContract --network <network>
 ```
-zos upgrade MyContract --network development
-```
-
-The address of the upgraded contracts is the same as before, but the code has been changed to the new
-version.
 
-_Voilà!_ You have deployed and upgraded an application using ZeppelinOS. 
+_Voilà_, we have deployed and upgraded an application using ZeppelinOS! The address of the upgraded contract is the same as before, but the code has been updated to the new version.
 
-If you want to use the ZeppelinOS standard libraries, please follow our [Using the stdlib in your app](using.md) guide. You can also check the API reference with [all the available commands of zos](climain.md).
+Next step is to [use the ZeppelinOS standard libraries](using.md) from our application.
```

Comment:
I'm ok with the you->we change, but this does not make clear that this next step is optional. We could go with:

In order to use the ZeppelinOS standard libraries in an upgradeable app, please follow our [next guide](using.md).